packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -729,7 +729,7 @@ describe('ReactHooks', () => {
     ReactTestRenderer.create(<App deps={undefined} />);
   });
 
-  it('assumes useEffect clean-up function is either a function or undefined', () => {
+  it('assumes useLayoutEffect clean-up function is either a function or undefined', () => {
```

Comment:
Ideally, the error message here that says "useEffect" would say "useLayoutEffect", since a layout effect is triggering the warning.